README.md

```diff
@@ -59,17 +40,16 @@ A Gutter Game is when the player never hits a pin (20 zero scores).
 
 A Perfect Game is when the player rolls 12 strikes (10 regular strikes and 2 strikes for the bonus in the 10th frame). The Perfect Game scores 300 points.
 
-In the image below you can find some score examples.
-
-More about ten pin bowling here: http://en.wikipedia.org/wiki/Ten-pin_bowling
-
-![Ten Pin Score Example](images/example_ten_pin_scoring.png)
-
-## Code Review
+### To Use This App
 
-In code review we'll be hoping to see:
+```git clone``` this repo
+then ```open index.html``` to see the app
+you can also run the jasmine tests by ```open SpecRunner.html```
```

Comment:
Great. Always useful when people put instructions on how to run the app in the README. I always forget. It's good practice.